Sources/LeafKit/LeafSerialize/ParameterResolver.swift

```diff
@@ -234,15 +246,16 @@ internal struct ParameterResolver {
                     rhs[key] = val
                 }
                 return .init(.dictionary(rhs))
-            case .null:
-                throw "unable to concatenate `null` with `\(rhs)'"
+                
+            case .optional(_, _): throw "Optional unwrapping not possible yet"
```

Comment:
Realistically this should never happen at resolve as it should decay prior, but hey.